pkg/download/blob.go

```diff
@@ -73,44 +74,53 @@ func NewBlobDownload(accountName, accountKey string, blob blobutil.AzureBlobRef)
 // GetSASBlob download a blob with specified uri and sas authorization and saves it to the target directory
 // Returns the filePath where the blob was downloaded
 func GetSASBlob(blobURI, blobSas, targetDir string) (string, error) {
-	bloburl, err := url.Parse(blobURI + blobSas)
+	blobFullURL := blobURI + blobSas
+
 	loggableBlobUri := GetUriForLogging(blobURI)
-	if err != nil {
-		return "", errors.Wrapf(err, "unable to parse URL: %q", loggableBlobUri)
-	}
 
-	containerRef, err := storage.GetContainerReferenceFromSASURI(*bloburl)
+	resp, err := http.Get(blobFullURL)
```

Comment:
Please sync with the latest unit test changes and make sure all tests still pass.

Reply:
sync'd . tests passing